### DISCUSSION TITLE
Welcome to django-whatsapp-web-clone Discussions!

### DISCUSSION THREAD

**Discussion by josnin** [codingelle/django-whatsapp-web-clone#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rohith-7** [comment]
is the code working now or any edits required??

**Reply by josnin** [reply]
its working now, just need to add more functionality @Rohith-7

**Comment by MigAru** [comment]
Mb i can help to you? 
I can make it so that the messages are saved in the database for later loading

**Reply by MigAru** [reply]
ok, this is not necessary
but, why you don't use pydantic with tortoise

**Reply by josnin** [reply]
Its already saving to database @MigAru 

This repo is actually focus on how to create chat using Django channel in our YouTube tutorial, then later decided to add more functionality like  saving msg to database, and decided to use tortoise 

There s not much validation checks since its only a simple functionality, so Pydantic is not yet use

Feel free to add more functionality and enhance @MigAru  if you have time

**Reply by MigAru** [reply]
i can do custom auth for the messenger)

sry my broken en(

**Reply by MigAru** [reply]
but i specialize on backend

**Reply by lordkadoz765** [reply]
@MigAru can you please make a saving messages to db please would really appreciate it

**Reply by MigAru** [reply]
@lordkadoz765, I didn't really understand the sentence, could you write in more detail what you want from me

**Reply by MigAru** [reply]
@lordkadoz765 do you want to make the message data saved to the database using tortoise orm? Did I understand you correctly?
